ecoscope_workflows/tasks/results/_ecoplot.py

```diff
@@ -82,21 +107,19 @@ def draw_time_series_bar_chart(
         Literal["year", "month", "week", "day", "hour"],
         Field(description="Sets the time interval of the x axis."),
     ],
-    groupby_style_kws: Annotated[
-        dict | SkipJsonSchema[None],
+    grouped_styles: Annotated[
+        list[GroupedPlotStyle],
         Field(
             description="Style arguments passed to plotly.graph_objects.Bar and applied to individual groups."
         ),
+    ] = [],
+    plot_style: Annotated[
+        PlotStyle | SkipJsonSchema[None],
+        Field(description="Additional style kwargs passed to go.Pie()."),
     ] = None,
-    style_kws: Annotated[
-        dict | SkipJsonSchema[None],
-        Field(
-            description="Style arguments passed to plotly.graph_objects.Bar and applied to all groups."
-        ),
-    ] = None,
-    layout_kws: Annotated[
-        dict | SkipJsonSchema[None],
-        Field(description="Style arguments passed to plotly.graph_objects.Figure."),
+    layout_style: Annotated[
+        LayoutStyle | SkipJsonSchema[None],
```

Comment:
```suggestion
        BarLayoutStyle | SkipJsonSchema[None],
```
Updating here per above suggestion